shared/lib/four-byte.ts

```diff
@@ -34,6 +34,10 @@ export async function getMethodFrom4Byte(
     functionName: 'getMethodFrom4Byte',
   })) as FourByteResponse;
 
+  if (!fourByteResponse.results || fourByteResponse.results.length === 0) {
```

Comment:
Minor, but should it be `!fourByteResponse.results?.length` as otherwise if `results` was set but was an object or invalid format without a `length`, then the below would still run?

Reply:
You are right, changed [here](https://github.com/MetaMask/metamask-extension/pull/27686/commits/c48141f0fc921e5bc4e90de9839f044a9905cff3).